config/manager/manager.yaml

```diff
@@ -37,9 +37,9 @@ spec:
           name: RELATED_IMAGE_csi-powermax
         - value: docker.io/dellemc/csipowermax-reverseproxy:v2.10.0
           name: RELATED_IMAGE_csipowermax-reverseproxy
-        - value: docker.io/dellemc/csi-powerstore:v2.11.0
+        - value: docker.io/dellemc/csi-powerstore:v2.11.1
```

Comment:
Same here.. Please update the version for CSM and dell-csm-operator in line no s 18 and 32 respectively